src/Umbraco.Web.BackOffice/Controllers/MemberController.cs

```diff
@@ -92,7 +96,8 @@ public class MemberController : ContentControllerBase
         IJsonSerializer jsonSerializer,
         IPasswordChanger<MemberIdentityUser> passwordChanger,
         ICoreScopeProvider scopeProvider,
-        ITwoFactorLoginService twoFactorLoginService)
+        ITwoFactorLoginService twoFactorLoginService,
+        IOptionsSnapshot<SecuritySettings>? securitySettings)
```

Comment:
Any reason for using `IOptionsSnapshot<>` instead of `IOptions<>` and making this nullable? Making it optional will result in different behaviour, where using the 'wrong' constructor will not honour the setting value.

Reply:
I have change this to IOption. I was not sure what was best to use here. I was looking at other controllers that were using IOptionSnapshot and mimicking what was done in those classes. IOptionSnapshot changed to IOption.